platform/pulp/platform/models/consumer.py

```diff
@@ -0,0 +1,67 @@
+"""
+Django models related to content consumers.
+"""
+
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db.models import TextField, ForeignKey, ManyToManyField, CASCADE
+
+from pulp.platform.models import Model, Notes
+from pulp.platform.models.repository import Distributor
```

Comment:
For consistency, Models should always be imported from `pulp.platform.models`, or from their actual location. So, this should either be:
```python
from pulp.platform.models import Model, Notes, Distributor
```
or
```python
from pulp.platform.models.base import Model
from pulp.platform.models.generic import Notes
from pulp.platform.models.repository import Distributor
```

We should decide which we prefer, commit to it, and get it documented in the translation guide.